string_permutation_problem/permute.py

```diff
@@ -0,0 +1,25 @@
+def permute(s):
+    out = []
+    
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpb1s7hhaw/string_permutation_problem/permute.py
+++ b/tmp/tmpb1s7hhaw/string_permutation_problem/permute.py
@@ -1,6 +1,6 @@
 def permute(s):
     out = []
-    
+
     # Base Case
     if len(s) == 1:
         out = [s]
```